client/components/mma/switch/SwitchOptions.tsx

```diff
@@ -163,7 +172,7 @@ export const SwitchOptions = () => {
 							You're currently supporting the Guardian with a{' '}
 							{monthlyOrAnnual.toLowerCase()} contribution of{' '}
 							{mainPlan.currency}
-							{currentAmount}.
+							{currentAmount.toFixed(2)}.
```

Comment:
this could also be an integer - seems worth checking all prices before doing toFixed(2)

Reply:
Yeah hence why I've popped this back into draft